Chapters/ChapterTwo.tex

```diff
@@ -46,14 +46,208 @@ \section{Robotics Frameworks}
+ \hspace{0.5cm} A competição Micromouse é um concurso anual na qual estudantes do mundo todo desenvolvem pequenos robôs autônomos, chamados \textit{micromouse}, postos a correr dentro de um labirinto. Dessa forma, o \textit{micromouse} que mais rápido chegar ao seu centro é o vencedor da competição.
+
+ \begin{figure}[H]
+ 	\centering
+ 	\includegraphics[width=0.5\textwidth]
+ 	{Figures/MoonlightSpecial.jpg}
+ 	\caption{\label{fig:MoonlightSpecial} Moonlight Special - Primeiro modelo \textit{micromouse} a ganhar uma competição.}
+ \end{figure}
+
+ \hspace{0.5cm} Sua ideia surge em 1977, quando a \textit{IEEE Spectrum Magazine} trouxe pela primeira vez o conceito de robôs autônomos para resolução de labirintos. Pouco tempo depois, sua primeira competição foi realizada, em junho de 1979, na primeira \textit{IEEE Amazing Micromouse Maze Contest} organizada na cidade de Nova York. Rapidamente, o conceito da competição se espalhou e, já no começo da década de 90, vários clubes voltados para Micromouse surgiam em escolas e universidades do mundo todo.\textbf{[From: The inception of Chedda] }
+ 
+ \hspace{0.5cm} Atualmente, a \textit{IEEE Micromouse Competition} adota uma configuração que consiste-se de um labirinto de 16 x 16 blocos. Cada bloco possui 18 cm x 18 cm. As paredes, que possuem 5 cm de altura, são pintadas de branco de modo a ser reflexiva à luz infravermelho. O chão, por outro lado, é pintado de preto, para que não seja reflexivo. Além disso, o \textit{micromouse} sempre inicia a partir de um dos cantos do labirintos e termina em seu centro. Com base nisso, os competidores devem usar de algoritimos de busca para explorar o labirinto para encontrar a rota mais otimizada para a resolução do labirinto. O robô por sua vez, não pode ter suas dimensões maiores que uma seção de 25cm x 25 cm. As regras completas estão dispostas como anexo no final do documento.
 
 
 %--------- NEW SECTION ----------------------
-\section{Assunto 1}
-\label{sec:ass1}
-flkjasdlkfjasdlkfjs
+\section{Benchmark}
+\label{sec:benchmark}
+
+\subsection{GreenGiant}
+\hspace{0.5cm} A Green Giant é uma desenvolvedora de múltiplas plataformas de robótica, especializada em eletrônica embarcada, tendo como seu carro-chefe o \textit{micromouse}. Seu modelo mais recente, 2016 - 2017, é voltado para alto desempenho em competições, alcançando a posição de quarto lugar durante a APEC de 2016.
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.7\textwidth]
+	{Figures/GreenGiant_model.jpg}
+	\caption{\label{fig:Green_Giant_model} Green Giant. }
+	\end{figure}
+
+\hspace{0.5cm}Sua interface de usuário possui \textit{Dot Matrix Display}, sinalizadores led, butões, buzzer, além de possuir um sistema de comunicação Bluetooth 4.0. Ademais, o modelo usa um sistema de ventoinhas de sucção para aumentar o nível de aderência das rodas, permitindo alcançar maiores velocidades sem derrapar.
+
+
+\begin{figure}[H]
```

Comment:
Isso não seria uma tabela não?

Se sim, para não fazer na mão, tem essa [ferramenta](https://www.tablesgenerator.com/) online caso precise. Se for tabela, lembrar de seguir ABNT, colocando título e fonte.

Reply:
O que seria uma tabela?